Tutorials/Editors-Manual/Getting-Started-With-Umbraco/index-v7.md

```diff
@@ -0,0 +1,43 @@
+---
+versionFrom: 7.0.0
+---
+
+# Getting started with Umbraco
+
+## [Logging In and Out](Logging-In-and-Out)
```

Comment:
This still links to the Umbraco 8 version 🙈

Reply:
have i misunderstood ? i thought it was supposed to link to the umbraco 8 version ?